auth0/src/main/java/com/auth0/android/authentication/storage/CryptoUtil.java

```diff
@@ -117,15 +125,14 @@ public CryptoUtil(@NonNull Context context, @NonNull Storage storage, @NonNull S
                         .setEndDate(end.getTime());
 
                 KeyguardManager kManager = (KeyguardManager) context.getSystemService(Context.KEYGUARD_SERVICE);
-                Intent authIntent = null;
                 if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.LOLLIPOP) {
                     //The next call can return null when the LockScreen is not configured
-                    authIntent = kManager.createConfirmDeviceCredentialIntent(null, null);
-                }
-                boolean keyguardEnabled = Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP && kManager.isKeyguardSecure() && authIntent != null;
```

Comment:
removed this check `Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP ` since it's already done a few lines above